packages/css/src/plugins/discard-empty-rules.ts

```diff
@@ -0,0 +1,25 @@
+import type { Plugin } from 'postcss';
```

Comment:
Just out of interest, since empty values are handled by babel-plugin, why do we need this plugin?

Reply:
Empty values still generate CSS and CSS classes (i.e. color: undefined) , so we need this plugin to remove them from the style sheet.